tests/backup/backup_portworx_test.go

```diff
@@ -562,3 +562,237 @@ var _ = Describe("{RestoreEncryptedAndNonEncryptedBackups}", func() {
+			dash.VerifyFatal(err, nil, "Creating source and destination cluster")
+			clusterStatus, _ = Inst().Backup.RegisterBackupCluster(orgID, SourceClusterName, "")
+			dash.VerifyFatal(clusterStatus, api.ClusterInfo_StatusInfo_Online, fmt.Sprintf("Verifying backup cluster %s", SourceClusterName))
+		})
+		for _, namespace := range appNamespaces {
+			for backupLocationUID, backupLocationName := range backupLocationMap {
+				Step("Getting size of volume before resizing", func() {
+					pods, err := core.Instance().GetPods(namespace, labelSelectors)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching the pod list"))
+					srcClusterConfigPath, err := GetSourceClusterConfigPath()
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Getting kubeconfig path for source cluster %v", srcClusterConfigPath))
+					for _, pod := range pods.Items {
+						beforeSize, err = getSizeOfMountPoint(pod.GetName(), namespace, srcClusterConfigPath)
+						dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching the size of volume before resizing %v from pod %v", beforeSize, pod.GetName()))
+						podListBeforeSizeMap[pod.Name] = beforeSize
+					}
+				})
+				Step("Create schedule policy", func() {
+					log.InfoD("Create schedule policy")
+					ctx, err := backup.GetAdminCtxFromSecret()
+					dash.VerifyFatal(err, nil, "Fetching px-central-admin ctx")
+					periodicSchedulePolicyName = fmt.Sprintf("%s-%v", "periodic", time.Now().Unix())
+					periodicSchedulePolicyNames = append(periodicSchedulePolicyNames, periodicSchedulePolicyName)
+					periodicSchedulePolicyUid = uuid.New()
+					periodicSchedulePolicyUids = append(periodicSchedulePolicyUids, periodicSchedulePolicyUid)
+					periodicSchedulePolicyInfo := Inst().Backup.CreateIntervalSchedulePolicy(5, 15, 5)
+					err = Inst().Backup.BackupSchedulePolicy(periodicSchedulePolicyName, periodicSchedulePolicyUid, orgID, periodicSchedulePolicyInfo)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of periodic schedule policy of interval 15 minutes named [%s]", periodicSchedulePolicyName))
+					periodicSchedulePolicyUid, err = Inst().Backup.GetSchedulePolicyUid(orgID, ctx, periodicSchedulePolicyName)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching uid of periodic schedule policy named [%s]", periodicSchedulePolicyName))
+				})
+				Step("Resize the volume before backup schedule", func() {
+					log.InfoD("Resize the volume before backup schedule")
+					for _, ctx := range contexts {
+						var appVolumes []*volume.Volume
+						log.InfoD(fmt.Sprintf("get volumes for %s app", ctx.App.Key))
+						appVolumes, err := Inst().S.GetVolumes(ctx)
+						dash.VerifyFatal(err, nil, fmt.Sprintf("Fetch volumes for app %s", ctx.App.Key))
+						dash.VerifyFatal(len(appVolumes) > 0, true, "Verifying if app volumes exist")
+						var requestedVols []*volume.Volume
+						log.InfoD(fmt.Sprintf("Increase volume size %s on app %s's volumes: %v",
+							Inst().V.String(), ctx.App.Key, appVolumes))
```

Comment:
Here it is going to print out the whole `appVolumes` struct. Print only what is required.

Reply:
removed appVolumes as not required to print